examples/index.html

```diff
@@ -6,6 +6,7 @@
   </head>
   <body>
     <div class="container py-4">
+      <p><a href="#">Add a Comment</a></p>
```

Comment:
Nit links should be links to somewhere and add a comment doesn't read like a link. Demo pages should avoid bad patterns so people can't copy/paste bad code.

```suggestion
      <p><button type="button">Add a Comment</button></p>
```